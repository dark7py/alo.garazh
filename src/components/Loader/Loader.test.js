import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import Loader from './Loader';

test('Modal does not crash', () => {
  const tree = shallow(
    <Loader />
  );

  expect(toJson(tree)).toMatchSnapshot();
});
