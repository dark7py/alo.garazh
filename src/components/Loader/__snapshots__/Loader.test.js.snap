// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Modal does not crash 1`] = `
<div
  className="loader"
>
  <div
    className="lds-ring"
  >
    <div />
    <div />
    <div />
    <div />
  </div>
</div>
`;
